firefox_ui_tests/firefox_ui_tests/functional/security/test_safe_browsing_primary_download.py

```diff
@@ -0,0 +1,69 @@
+import sys
+import os
+from marionette_driver import By, Wait
+from marionette.marionette_test import skip_if_e10s
+
+from firefox_puppeteer.testcases import FirefoxTestCase
+
+from firefox_puppeteer.ui.browser.window import BrowserWindow
+
+
+class TestSafeBrowsingPrimaryDownload(FirefoxTestCase):
+
+    file_names = [
+        "goog-phish-shavar.cache",
+        "goog-phish-shavar.pset",
+        "goog-phish-shavar.sbstore",
+        "goog-malware-shavar.cache",
+        "goog-malware-shavar.pset",
+        "goog-malware-shavar.sbstore",
+        "goog-badbinurl-shavar.cache",
+        "goog-badbinurl-shavar.pset",
+        "goog-badbinurl-shavar.sbstore"
+    ]
+
+    win_file_names = [
+        "goog-downloadwhite-digest256.cache",
+        "goog-downloadwhite-digest256.pset",
+        "goog-downloadwhite-digest256.sbstore"
+    ]
+
+    tracking_protection = [
+        "mozstd-track-digest256.cache",
+        "mozstd-track-digest256.pset",
+        "mozstd-track-digest256.sbstore",
+        "mozstd-trackwhite-digest256.cache",
+        "mozstd-trackwhite-digest256.pset",
+        "mozstd-trackwhite-digest256.sbstore"
+    ]
```

Comment:
Personally I would make one dictionary with all those file names. It can be separated by type of protection and limitation for platforms.